src/pages/settings/bank-accounts/create/Create.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Card, Element } from '@invoiceninja/cards';
+import { InputField } from '@invoiceninja/forms';
+import { endpoint } from 'common/helpers';
+import { request } from 'common/helpers/request';
+import { route } from 'common/helpers/route';
+import { useTitle } from 'common/hooks/useTitle';
+import { useFormik } from 'formik';
+import { useRef, useState } from 'react';
+import toast from 'react-hot-toast';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { Settings } from '../../../../components/layouts/Settings';
+import { useBankAccountPages } from '../common/hooks/useBankAccountPages';
+import { BankAccountValidation } from '../common/validation/ValidationInterface';
+
+const Create = () => {
+  useTitle('create_bank_account');
+  const [t] = useTranslation();
+  const navigate = useNavigate();
+  const pages = useBankAccountPages();
+  const bankAccountForm = useRef<any>();
+  const [isFormBusy, setIsFormBusy] = useState<boolean>(false);
+  const [errors, setErrors] = useState<BankAccountValidation | undefined>(
+    undefined
+  );
+
+  const form = useFormik({
+    enableReinitialize: true,
+    initialValues: {
+      bank_account_name: '',
+    },
+    onSubmit: async (values: any) => {
```

Comment:
We are using TypeScript for a reason. Please don't make use of `any` unless is absolutely needed. If you don't know the value (which you should always do, unless writing a return point or interface declaration) use `unknown` instead of `any`.